easy/16.js

```diff
@@ -1,5 +1,7 @@
 function charIndex(word, char) {
-	
+    if (word.indexOf(char) > -1 && word.lastIndexOf(char) > -1) {
+        return [word.indexOf(char), word.lastIndexOf(char)];
+    } else return undefined;

```

Comment:
You can skip else. When `if` would be fulfilled then return array and next part of code won't be executed. You can just type `return` after curly brackets in `if`